### PR TITLE
Shekhar/prod script bg images

### PR DESCRIPTION
**Pull request by shekharkhedekar**
https://3.basecamp.com/5067876/buckets/22032865/todos/5883136001

### CODE REVIEW

**Review by mrengy — CHANGES_REQUESTED** [review]
Thanks for this. 

### Background

I'm not seeing the distress image rendered. I do see the distress image in the CSS as another url for `background-image`. Only visible difference I can tell is that the photo is more compressed - looks a bit blurrier and lower contrast. 

Screenshot 2023-04-12 at 4 09 49 PM

Looking at this - [MDN: Using multiple backgrounds](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Backgrounds_and_Borders/Using_multiple_backgrounds), I think you need to swap the order of the distress image and the background photo in the CSS in order for them to layer correctly (distress on top of photo).

Also I thought we talked about applying a solid background color (red or green) over the photo, so the desired layer order would be (top to bottom):

1. Distress image
2. Solid background color
3. Background photo

Unfortunately from that MDN article, I saw

> You can apply multiple backgrounds to elements. These are layered atop one another with the first background you provide on top and the last background listed in the back. **Only the last background can include a background color**.

so it might require additional divs? If that gets gross, we don't have to include all those layers. I'll let you sort out the approach.

### Prod search and replace script

I ran `npm run replace-urls:production` and got a bunch of warnings like 

> Deprecated: Creation of dynamic property WP_CLI\Dispatcher\CompositeCommand::$longdesc is deprecated in phar:///usr/local/Cellar/wp-cli/2.7.1/bin/wp/vendor/wp-cli/wp-cli/php/WP_CLI/Dispatcher/CompositeCommand.php on line 38

but I think that's just because I'm using an old version of WP CLI locally, and I think it succeeded anyway.

Looks as though you had the prod url wrong ;) I know the convention is not consistent between stage and production. I think I fixed it with f4232c14f8dd781c3ae494d9118d0f82d9cba01e , but I didn't test this, so check it on your end.

**Comment by shekharkhedekar** [comment]
Yeah, noticed the wrong url too and forgot to push it. That didn't seem to fix it.

RE: the backgrounds, I'll switch the order. For the color, what do you think about a third image that is the overlay between the distress and photo?

**Comment by mrengy** [comment]
Oh, okay. Hm, not sure on the url then. Maybe there’s a history.scvanguard.org (without the https://) or something in the db? Try wp search-replace to find variations like that? You can also pass —dry-run to the command to just show what replacements it would make without actually making them. 

Good idea on the background color. Maybe a tiled 1px png could work?

--
Mike Eng (he/him)
mike-eng.com linkedin.com/in/mikeeng401

File: README.md [hunk 1]
```diff
@@ -85,7 +85,10 @@ If you get `Error: Error establishing a database connection`, you'll need to edi
 
 If in the command line, you see an error involving `gravityformscli` or `wp gf`, there was likely a problem with the script that installs the GravityForms plugin. You can try to install it by using one of the two following methods:
 
-1. In the command line, run `wp gf install --key=283af92393f339347c5e7f6aa889e0cc`
+1. In the command line, run:
+   ```sh
```
Comment (shekharkhedekar) [inline README.md:89]: easier copy paste

File: wp-content/themes/vanguard-history/css/_background.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+@media screen and (min-width: 1280px) {
+	body#background-1 {
+		background-image: url("img/distress-2.png"),
+			url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mNUDDeqBwACdgErtXLrFQAAAABJRU5ErkJggg==),
+			url("img/background-1.png");
+	}
+
+	body#background-2 {
+		background-image: url("img/distress-2.png"),
+			url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mNUDDeqBwACdgErtXLrFQAAAABJRU5ErkJggg==),
```
Comment (shekharkhedekar) [inline wp-content/themes/vanguard-history/css/_background.scss:14]: ended up using base64 1x1 images generated here https://png-pixel.com/

**Comment by shekharkhedekar** [comment]
btw it ended up working when I hit `https://vanguard-history.local` - not sure why it required https but it works

**Review by mrengy — APPROVED** [review]

**Comment by mrengy** [comment]
Hm, maybe something in your .htaccess file, but I'm not sure off-hand how to resolve it. Or perhaps you should change the replace-urls scripts and readme documentation to replace the url to https://vanguard-history.local ? 

For what it's worth, my local works when going to http://localhost:10003  without https. There's some reason I had to use the port in there like that, but that's a separate thing

**Comment by mrengy** [comment]
Trying to use https:// on my local is giving me a `Secure Connection Failed` error in the browser, actually, so probably not best to change the script to replace to https. I'd assume this is a local config thing on your end.

**Pull request merged**